Tasks/PackerBuildV1/src/packerHost.ts

```diff
@@ -101,12 +101,22 @@ export default class PackerHost implements definitions.IPackerHost {
         var installedPackerPath = tl.which("packer", false);
         var installedPackerVersion = this._getPackerVersion(installedPackerPath);
         console.log(tl.loc("InstalledPackerVersion", installedPackerVersion));
-        if(!installedPackerVersion || 
-            utils.isGreaterVersion(utils.PackerVersion.convertFromString(constants.CurrentSupportedPackerVersionString), utils.PackerVersion.convertFromString(installedPackerVersion))) {
+        var packerVersionString = constants.CurrentSupportedPackerVersionString;
+        var explicitPackerVersion : boolean= false ;
 
-            console.log(tl.loc("DownloadingPackerRequired", constants.CurrentSupportedPackerVersionString, constants.CurrentSupportedPackerVersionString));
+        if(this._taskParameters.templateType == constants.TemplateTypeCustom && this._taskParameters.packerVersionString) {
+            packerVersionString = this._taskParameters.packerVersionString;
+            explicitPackerVersion = true;
```

Comment:
if installed packer version is same as explicit version, then we need to download again. 
Otherwise, do not detect installed version if explicit version is specified

Reply:
Added check to not consider explicit version if the installed version is same as the explicit version